contracts/pl.json

```diff
@@ -0,0 +1,37 @@
+{
+  "generic_goodluck": "Powodzenia!",
+  "generic_objectives": "Aby zawrzeć tę umowę, musisz:",
```

Comment:
Can you confirm this is the right verb to mean "fulfill" or "complete" ?

Reply:
Okay when it comes to Polish - the context is important. I basically wrote "to sign" or "to accept" this contract. This has been changed to "complete" the contract